aiidalab/app.py

```diff
@@ -523,7 +530,7 @@ class AiidaLabAppWatch:
             The AiidaLab app to monitor.
     """
 
-    class AppPathFileSystemEventHandler(FileSystemEventHandler):
+    class AppPathFileSystemEventHandler(FileSystemEventHandler):  # type: ignore
```

Comment:
Why is this ignore needed?

Reply:
It complains about "error: Class cannot subclass "FileSystemEventHandler" (has type "Any")  [misc]", I simply suppress it for the moment.